x-pack/test/security_solution_api_integration/test_suites/detections_response/rules_management/prebuilt_rules/management/trial_license_complete_tier/upgrade_review_prebuilt_rules.ts

```diff
@@ -0,0 +1,679 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+import expect from 'expect';
+import {
+  ThreeWayDiffOutcome,
+  ThreeWayMergeOutcome,
+} from '@kbn/security-solution-plugin/common/api/detection_engine';
+import { FtrProviderContext } from '../../../../../../ftr_provider_context';
+import {
+  deleteAllTimelines,
+  deleteAllPrebuiltRuleAssets,
+  createRuleAssetSavedObject,
+  installPrebuiltRules,
+  createPrebuiltRuleAssetSavedObjects,
+  reviewPrebuiltRulesToUpgrade,
+  patchRule,
+  createHistoricalPrebuiltRuleAssetSavedObjects,
+} from '../../../../utils';
+import { deleteAllRules } from '../../../../../../../common/utils/security_solution';
+
+export default ({ getService }: FtrProviderContext): void => {
+  const es = getService('es');
+  const supertest = getService('supertest');
+  const log = getService('log');
+
+  describe('@ess @serverless @skipInServerlessMKI review prebuilt rules updates from package with mock rule assets', () => {
+    beforeEach(async () => {
+      await deleteAllRules(supertest, log);
+      await deleteAllTimelines(es, log);
+      await deleteAllPrebuiltRuleAssets(es, log);
+    });
+
+    describe(`single line string fields`, () => {
+      const getRuleAssetSavedObjects = () => [
+        createRuleAssetSavedObject({ rule_id: 'rule-1', version: 1, name: 'A' }),
+      ];
+
+      describe("when rule field doesn't have an update and has no custom value", () => {
+        it('should not show in the upgrade/_review API response', async () => {
+          // Install all prebuilt detection rules
+          await createHistoricalPrebuiltRuleAssetSavedObjects(es, getRuleAssetSavedObjects());
+          await installPrebuiltRules(es, supertest);
+
+          // Call the upgrade review prebuilt rules endpoint and check that no rules are eligible for update
+          const reviewResponse = await reviewPrebuiltRulesToUpgrade(supertest);
+          expect(reviewResponse.rules).toEqual([]);
```

Comment:
Test plan mentions that there needs to be a new version available for the rule. Perhaps we can test this scenario the same way as other scenarios - creating a new version of a rule that has only "version" field bumped, then checking that "name" is not in the list of updated fields.